docs/examples/validation-result.md

```diff
@@ -0,0 +1,77 @@
+# Examples: Validation Result
+
+## YAML
+
+### Example Command
+`./bin/go-oscal validate -f testdata/validation/invalid-catalog.yaml -r test_out/validation-result.yaml`
+
+### Output:
+
+```yaml
```

Comment:
I like that this addition works for both success and failure scenarios. Do you see a need for any information that relates a validation result to some source document?

Reply:
I have been back and forth on it, I think it may be a good Idea to toss in the filepath of the document being read. Ill add that and re-request.

Reply:
Resolved by 4af5af8efc6ed3f3b7eec0f1c025e5d01c56be03